_apps/minimum-reviews.md

```diff
@@ -0,0 +1,53 @@
+---
+# A human-friendly name of your listing
+title: Minimum reviews
+# A short description of what your app does
+description: Enforce a minimum number of reviews on Pull Requests
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: n/a
```

Comment:
This should be `minimum-reviews` if https://github.com/apps/minimum-reviews is your app?